### ISSUE TITLE
runtime error when NODE_ENV === 'production', 

### ISSUE THREAD

**Issue by iambumblehead** [nanostores/i18n#21]
https://github.com/nanostores/i18n/blob/main/create-i18n/index.js#L23-L41

this is the stacktrace that appears in my service
```bash
TypeError: Cannot read properties of undefined (reading 'split')
    at file:///app/node_modules/@nanostores/i18n/create-i18n/index.js:68:34
    at listener (file:///app/node_modules/nanostores/lifecycle/index.js:129:19)
    at object.events.<computed>.reduceRight.shared (file:///app/node_modules/nanostores/lifecycle/index.js:17:58)
    at Array.reduceRight (<anonymous>)
    at file:///app/node_modules/nanostores/lifecycle/index.js:17:31
    at Object.store.listen (file:///app/node_modules/nanostores/lifecycle/index.js:136:9)
```

here's the nanostores location where the error occurs https://github.com/nanostores/i18n/blob/main/create-i18n/index.js#L68

the error appears to be a consequence of this failing condition. if this condition is not true, `t.component` is not defined and runtime error occurs when `t.component.split('/')[0]` is used later in the script
```javascript
    if (process.env.NODE_ENV !== 'production') {
      t.component = componentName
      t.base = base
      if (define.cache[baseLocale][componentName]) {
        if (import.meta && (import.meta.hot || import.meta.webpackHot)) {
          /* c8 ignore next 3 */
          for (let i of define.cache) {
            delete define.cache[i][componentName]
          }
        } else {
          // eslint-disable-next-line no-console
          console.warn(
            `I18n component ${componentName} was defined multiple times. ` +
              'It could lead to cache issues. Try to move i18n definition ' +
              'from component’s render function.'
          )
        }
      }
    }
```

**Comment by ai** [comment]
Yes. Wait a second.

**Comment by ai** [comment]
Fixed 7728df9

**Comment by iambumblehead** [comment]
@ai please publish this to npm for me :)

**Comment by ai** [comment]
Of course, on the way =^_^=

**Comment by ai** [comment]
The fix was released in 0.5.1

**Comment by iambumblehead** [comment]
thank you
